minetest.conf.example

```diff
@@ -96,8 +96,8 @@
 #    type: bool
 # fixed_virtual_joystick = false
 
-#    (Android) Use virtual joystick to trigger "aux" button.
-#    If enabled, virtual joystick will also tap "aux" button when out of main circle.
+#    (Android) Use virtual joystick to trigger "AUX" button.
+#    If enabled, virtual joystick will also tap "AUX" button when out of main circle.
```

Comment:
You don't need to edit this file, it is auto-generated from settingtypes.txt.
Also, 'AUX' is wrong as explained.